Eatery/Views/Eateries/HomescreenViews/MenuCollectionViewHeaderView.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  MenuCollectionViewHeaderView.swift
+//  Eatery
+//
+//  Created by Noah Pikielny on 4/19/21.
+//  Copyright © 2021 Cornell AppDev. All rights reserved.
+//
+
+import UIKit
+
+class MenuCollectionViewHeaderView: UICollectionReusableView {
+
+    let titleLabel = UILabel()
```

Comment:
private let

Reply:
I copied conventions from another header view class in the project that allows the table view delegate to access the title label. Should I change them both to private and add a configure function or should I leave them as is?